user_docs/en/userGuide.t2t

```diff
@@ -2006,15 +2006,15 @@ Please see the display's documentation for descriptions of where these keys can
 | Report title | etouch2 |
 | Report status bar | etouch4 |
 | shift+tab key | sp1 |
-| alt key | sp2 |
+| alt key | sp2, alt |
 | escape key | sp3 |
 | tab key | sp4 |
 | upArrow key | spUp |
 | downArrow key | spDown |
 | leftArrow key | spLeft |
 | rightArrow key | spRight |
-| enter key | spEnter, enter|
-| Report date/time | sp2+sp3 |
+| enter key | spEnter, enter |
+| Report date/time | sp1+sp2 |

```

Comment:
Could you please revert this change?

Reply:
@leonardder Done. I'm a real newbie to git, so i might have done somethiing wrong, but it looks like it is working.